libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -16,7 +16,7 @@
 // position controller default definitions
 #define POSCONTROL_ACCELERATION_MIN             50.0f   // minimum horizontal acceleration in cm/s/s - used for sanity checking acceleration in leash length calculation
 #define POSCONTROL_ACCEL_XY                     100.0f  // default horizontal acceleration in cm/s/s.  This is overwritten by waypoint and loiter controllers
-#define POSCONTROL_ACCEL_XY_MAX                 980.0f  // max horizontal acceleration in cm/s/s that the position velocity controller will ask from the lower accel controller
+#define POSCONTROL_ACCEL_XY_MAX                 (GRAVITY_MSS*100)  // max horizontal acceleration in cm/s/s that the position velocity controller will ask from the lower accel controller
```

Comment:
Is there a good reason to limit xy max to gravity?  This looks to be historical coincidence as a value.

Reply:
@peterbarker, I think this definition leads to a lean angle limit of 45deg.  I'm not sure it's worth changing though.